### DISCUSSION TITLE
PaddleOCR/latest/ppocr/model_list

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#14151] [Q&A]
# PaddleOCR/latest/ppocr/model_list

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/latest/ppocr/model_list.html

**Comment by AIwzx** [comment]
奈斯

**Comment by happyhaix** [comment]
very good models,make me brain spin. love it

**Reply by funnymdpee** [reply]
非常好的模型，使我的大脑旋转，爱来自hh

**Comment by mumutu66** [comment]
typo `ch_PP-OCRv4_rec_distill.yml` ->  `ch_PP-OCRv4_rec_distillation.yml`

**Reply by yedemon** [reply]
just what i need.

**Comment by haotianmingyue** [comment]
ch_PP-OCRv4_server_rec	对应的配置文件ch_PP-OCRv4_rec_hgnet.yml没有了, 它现在对应那个配置文件?

**Reply by haotianmingyue** [reply]
模型对应这个配置文件PP-OCRv4_server_rec.yml.
